remoteappmanager/handlers/home_handler.py

```diff
@@ -265,8 +265,12 @@ def _start_container(self, orm_user, app, policy, mapping_id):
         volumes = {}
 
         if mount_home:
-            home_path = os.path.expanduser('~'+user_name)
-            volumes[home_path] = {'bind': '/workspace', 'mode': 'rw'}
+            home_path = os.environ.get('HOME')
```

Comment:
Let me check. If it's the shell that sets it, then we are in the clear, provided jupyterhub is invoking the process without a shell.